modules/ROOT/pages/release-notes.adoc

```diff
@@ -4,4 +4,25 @@
 [NOTE]
 ====
 The 2023.1 is in development.
-====
\ No newline at end of file
+====
+
+== Feature deprecations and removals
+
+=== Removals
+
+==== Multi-tenancy removal
+
+Deprecated since Bonita 2021.1, xref:2022.2@ROOT:multi-tenancy-and-tenant-configuration.adoc[multi-tenancy] feature has been removed technically.
```

Comment:
add mention on Spring objects that changed / were removed, so that if a customer has used internal spring stuffs, they are aware that some unexpected behavior can occur.

Reply:
too many objects have been changed/removed, adding them in the release note will pollute it.